dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -127,8 +128,8 @@ const containerStylesUntilLeftCol = css`
 	display: grid;
 
 	grid-template-rows:
-		[headline-start show-hide-start] auto
-		[show-hide-end headline-end content-toggleable-start content-start] auto
+		[headline-start controls-start] auto
```

Comment:
Now that the show-hide grid area contains show hide and nav buttons ive renamed this to `controls`